test_infra/src/lib.rs

```diff
@@ -82,6 +82,8 @@ impl GuestNetworkConfig {
 
             // Reply on epoll w/ timeout to wait for guest connections faithfully
             let epoll_fd = epoll::create(true).expect("Cannot create epoll fd");
+            // Use 'File' to enforce closing on 'epoll_fd'
+            let _epoll_file = unsafe { fs::File::from_raw_fd(epoll_fd) };
```

Comment:
It could have been slightly better if after creating the `epoll_file` you were actually using it. This helps understand the lifetime required by this variable.
You would do `epoll_file.as_raw_fd()` instead of `epoll_fd`.